apps/docs/components/title/title.css

```diff
@@ -7,27 +7,29 @@
 }
 
 :where(.hd-title--level1) {
-    font-size: 2.5em;
+    font-size: 2.25rem;
```

Comment:
shouldn't we have variables for all the font sizes?

Reply:
Yes, I'm planning to revamp the font-size tokens, actually most tokens, previously created, that were not really suitable for our use cases.